src/ZoweUSSNode.ts

```diff
@@ -12,11 +12,21 @@
 import * as zowe from "@brightside/core";
 import { Session } from "@brightside/imperative";
 import * as vscode from "vscode";
+// tslint:disable-next-line: no-duplicate-imports
+import { TreeItemCollapsibleState } from "vscode";
```

Comment:
This line is redundant, because `TreeItemCollapsibleState` is never used directly.